exist-docker/src/main/resources-filtered/Dockerfile

```diff
@@ -85,7 +67,7 @@ ENV JAVA_TOOL_OPTIONS \
   -XX:+UseStringDeduplication \
```

Comment:
Isn't `-XX:+UseStringDeduplication` invalid on JDK 17?

Reply:
This option is still valid see the [JDK 24](https://docs.oracle.com/en/java/javase/23/docs/specs/man/java.html)  Java command options, but only useful in conjunction with the G1 garbage collector... (`-XX:+UseG1GC`)

Reply:
> @reinhapa Yes you are correct. However, we are using the ZGC garbage collector, and it is not supported there and prints a warning.

@adamretter in this case the option `-XX:+UseStringDeduplication` is indeed useless and will be ignored...

Reply:
Yes with ZGC the setting is ignored, just as when you introduced the switch to ZGC. This PR does not change the JAVA_OPTIONS. It only addresses the outdated syntax warning from the Dockerfile. Revisiting the JAVA_OPTIONS makes more sense, with the PR for java21